native/c/zowex.cpp

```diff
@@ -536,19 +546,41 @@ int main(int argc, char *argv[])
 
   arg_parser.get_root_command().add_command(job_cmd);
 
-  // Check for interactive mode before parsing to avoid help text
+  // Version command
+  auto version_cmd = command_ptr(new Command("version", "display version information"));
+  version_cmd->add_alias("--version");
+  version_cmd->add_alias("-v");
+  version_cmd->set_handler(handle_version);
+  arg_parser.get_root_command().add_command(version_cmd);
+
+  // Check for version or interactive mode before parsing to avoid help text
   bool is_interactive = false;
+  bool is_version = false;
   for (int i = 1; i < argc; i++)
   {
     if (strcmp(argv[i], "--interactive") == 0 || strcmp(argv[i], "--it") == 0)
     {
       is_interactive = true;
       break;
     }
+    else if (strcmp(argv[i], "--version") == 0 || strcmp(argv[i], "-v") == 0)
+    {
+      is_version = true;
```

Comment:
So, whichever one gets specified first wins? 😅

Reply:
cfdc486 lets you do:
- `./zowex --it -v`
- `./zowex -v --it`